<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Utils</class>
 <widget class="QWidget" name="Utils">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>82</width>
     <height>146</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Аксана</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="aksa">
      <property name="text">
       <string>Ок</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinAxaX"/>
    </item>
    <item>
     <widget class="QSpinBox" name="spinAxaY"/>
    </item>
    <item>
     <widget class="QSpinBox" name="spinAxaZ"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>510</y>
     <width>166</width>
     <height>93</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Perspeck</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="persp">
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSpinBox" name="spinPPCount">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinPZL">
      <property name="minimum">
       <number>-10002</number>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>220</width>
     <height>125</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Kosa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="kosa">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="spinKL">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinKA">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>90</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>114</width>
     <height>177</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Взгляд</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="view">
        <property name="text">
         <string>Ок</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSpinBox" name="spinViewR">
      <property name="minimum">
       <number>-1000</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinViewF">
      <property name="minimum">
       <number>-1000</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinViewT">
      <property name="minimum">
       <number>-1000</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonView">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>102</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Поворот</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinPovorotX">
            <property name="minimum">
             <number>-300</number>
            </property>
            <property name="maximum">
             <number>300</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinPovorotY">
            <property name="minimum">
             <number>-300</number>
            </property>
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="value">
             <number>90</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinPovorotZ">
            <property name="minimum">
             <number>-300</number>
            </property>
            <property name="maximum">
             <number>300</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Перенос</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinMoveX">
            <property name="minimum">
             <number>-3000</number>
            </property>
            <property name="maximum">
             <number>3000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinMoveY">
            <property name="minimum">
             <number>-3002</number>
            </property>
            <property name="maximum">
             <number>3000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinMoveZ">
            <property name="minimum">
             <number>-10000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Масштабирование </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinScaleX">
            <property name="minimum">
             <number>-1000</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinScaleY">
            <property name="minimum">
             <number>-1000</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinScaleZ">
            <property name="minimum">
             <number>-1000</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="mrs">
      <property name="text">
       <string>Ок</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>21</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>R</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>F</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>T</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mrs</sender>
   <signal>clicked()</signal>
   <receiver>Utils</receiver>
   <slot>checkRotate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mrs</sender>
   <signal>clicked()</signal>
   <receiver>Utils</receiver>
   <slot>checkScale()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mrs</sender>
   <signal>clicked()</signal>
   <receiver>Utils</receiver>
   <slot>checkMove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aksa</sender>
   <signal>clicked()</signal>
   <receiver>Utils</receiver>
   <slot>checkAxa()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kosa</sender>
   <signal>clicked()</signal>
   <receiver>Utils</receiver>
   <slot>checkKosa()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>persp</sender>
   <signal>clicked()</signal>
   <receiver>Utils</receiver>
   <slot>checkPerspeck()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>view</sender>
   <signal>clicked()</signal>
   <receiver>Utils</receiver>
   <slot>checkView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkRotate()</slot>
  <slot>checkMove()</slot>
  <slot>checkScale()</slot>
  <slot>checkAxa()</slot>
  <slot>checkKosa()</slot>
  <slot>checkPerspeck()</slot>
  <slot>checkView()</slot>
 </slots>
</ui>
